### PR TITLE
add a custom triangle draw example

### PR DESCRIPTION
**Pull request by jeske**
The link to custom index / vertex buffer drawing didn't explain how to use MeshDraw

### CODE REVIEW

File: en/manual/scripts/create-a-model-from-code.md [hunk 1]
```diff
@@ -87,8 +87,36 @@ You can create models in scripts at runtime. You can do this in several differen
     model.Meshes.Add(mesh);
     ```
 
-    >[!Note]
-    >For information about how to set up vertex and index buffers, see [Drawing vertices](../graphics/low-level-api/draw-vertices.md).
+5. Here is a more complete example that draws a custom triangle..
+
+    ```cs
+    var verticies = new VertexPositionTexture[3];
```
Comment (Kryptos-FR) [inline en/manual/scripts/create-a-model-from-code.md:93]: Typo: plural of vertex is *vertices*.

File: en/manual/scripts/create-a-model-from-code.md [hunk 2]
```diff
@@ -87,8 +87,36 @@ You can create models in scripts at runtime. You can do this in several differen
...
+        IndexBuffer = new IndexBufferBinding(indexBuffer, true, indicies.Length),
+        VertexBuffers = new[] { new VertexBufferBinding(vertexBuffer, 
+	                                   VertexPositionTexture.Layout, vertexBuffer.ElementCount) },
+      } };            
+    // add the mesh to the model
+    model.Meshes.Add(customMesh);
+   
+    ```
+
+
```
Comment (Kryptos-FR) [inline en/manual/scripts/create-a-model-from-code.md:117]: Superfluous empty line can be removed.

File: en/manual/scripts/create-a-model-from-code.md [hunk 3]
```diff
@@ -87,8 +87,36 @@ You can create models in scripts at runtime. You can do this in several differen
...
+    ```cs
+    var verticies = new VertexPositionTexture[3];
+    verticies[0].Position = new Vector3(0f,0f,1f);            
+    verticies[1].Position = new Vector3(0f,1f,0f);
+    verticies[2].Position = new Vector3(0f,1f,1f);
+    var vertexBuffer = 
+        Stride.Graphics.Buffer.Vertex.New(GraphicsDevice, 
+	                                  verticies, GraphicsResourceUsage.Dynamic);
+    int[] indicies = { 0, 2, 1 };
```
Comment (Kryptos-FR) [inline en/manual/scripts/create-a-model-from-code.md:100]: Typo: plural of index is *indices*.

**Comment by jeske** [comment]
I fixed these changes, but accidentally merged this and the fixes into my other branch/pull-request.. so im going to close this.

https://github.com/stride3d/stride-docs/pull/36

**Pull request closed without merging**